exporter/prometheusexporter/collector.go

```diff
@@ -227,6 +235,83 @@ func (c *collector) convertDoubleHistogram(metric pmetric.Metric, resourceAttrs
 	return m, nil
 }
 
+func (c *collector) createTargetInfoMetrics(resourceAttrs []pcommon.Map) ([]prometheus.Metric, error) {
+	var metrics []prometheus.Metric
+	var lastErr error
+
+	// deduplicate resourceAttrs by job and instance
+	deduplicatedResourceAttrs := make([]pcommon.Map, 0, len(resourceAttrs))
```

Comment:
I think it should be safe to assume resource attributes are already deduplicated.  We assume uniqueness for metrics, so resource should also be OK IMO.

Reply:
No, so, lets say you have 100 metrics, 50 from one resource, and 50 from another. The `resourceAttrs` array will have 100 items, one for each metric.

But we should only export 2 `target_info` hence the deduplication.